appservice/src/createAppService/createAppService.ts

```diff
@@ -71,6 +71,8 @@ export async function createAppService(
             break;
         case AppKind.app:
             if (createOptions.advancedCreation) {
+                actionContext.properties.advancedCreation = 'true';
```

Comment:
Looks fine, although consider how you will distinguish between advancedCreation being undefined because it occurred before this checkin vs it wasn't advanced creation. Personally I'd consider 'false'.